includes/class-wc-payments-account.php

```diff
@@ -643,4 +646,53 @@ private function settings_changed( $changes = [] ) {
 		$diff = array_diff_assoc( $changes, $account );
 		return ! empty( $diff );
 	}
+
+	/**
+	 * Returns an array containing the names of all the WCPay related notes that have be actioned.
+	 *
+	 * @return array
+	 */
+	private function get_actioned_notes() {
```

Comment:
Can we also declare a return type here (`private function get_actioned_notes(): array`)?

Reply:
We can, thanks.